alphagov/govuk-coronavirus-business-volunteer-form
# Schema expectation tests

Trello: https://trello.com/c/FEZ30Fa2

## What's changed?

Creates a JSON schema for the FormSubmission model and adds tests to ensure that user form submissions are valid against the schema.

## Why

Currently we need to email Cabinet Office and their contractors when we make a change to the form. This isn't a sustainable process, so instead we'd like an automated test to ensure that we don't break things for folks downstream who are using the data we're generating.

## How to test locally

1. Run the tests locally
2. Run the app locally and submit the form. The form should submit successfully. 
3. There should be no changes to the user's experience.
